currencies.json

```diff
@@ -714,8 +714,8 @@
     "latestVersion": "",
     "cloudNative": false,
     "isBeta": false,
-    "ignoreUpdates": true,
-    "note": "[Otel](https://jsw.ibm.com/browse/INSTA-1561)",
+    "ignoreUpdates": false,
```

Comment:
Could we temporarily ignore the updates?

Reply:
Why? 🤔